src/screens/History/MultiChainHistoryEtherspot.js

```diff
@@ -58,11 +58,11 @@ function MultiChainHistoryEtherspot() {
   ];
 
   const items = chains.map((chain) => {
-    const { title } = chainsConfig[chain];
+    const { titleShort } = chainsConfig[chain];
 
     return {
       key: chain,
-      title,
+      title: titleShort,
```

Comment:
👀

Reply:
I've changed it because I've noticed that designs use short chain names, e.g. https://app.zeplin.io/project/5fa95ca539e6804a10dec0a4/screen/60b8c03675cc2d95be58958b